packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -4021,7 +4028,7 @@ abstract class Element extends DiagnosticableTree implements BuildContext {
     assert(depth != null);
     assert(owner != null);
     // Use the private property to avoid a CastError during hot reload.
```

Comment:
is this comment still accurate?

Reply:
There may be Element subtypes that still have an override w/ cast outside of the framework